### PR TITLE
khalid/removed undefined header

### PR DESCRIPTION
**Pull request by Khalidxx**

### CODE REVIEW

File: src/javascript/app/base/binary_loader.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ const BinaryLoader = (() => {
             return;
         }
 
-        const div_container = createElement('div', { class: 'logged_out_title_container', html: content.getElementsByTagName('h1')[0] });
+        const div_container = createElement('div', { class: 'logged_out_title_container', html: content.getElementsByTagName('h1')[0] ? content.getElementsByTagName('h1')[0] : '' });
```
Comment (negarn) [inline src/javascript/app/base/binary_loader.js]: better store it in a variable so we don't need to retrieve the element twice

Comment (negarn) [reply src/javascript/app/base/binary_loader.js]: or use `content.getElementsByTagName('h1')[0] || ''`

Comment (Khalidxx) [reply src/javascript/app/base/binary_loader.js]: Okay 👍

File: src/javascript/app/base/binary_loader.js [hunk 2]
```diff
@@ -149,7 +149,8 @@ const BinaryLoader = (() => {
             return;
         }
 
-        const div_container = createElement('div', { class: 'logged_out_title_container', html: content.getElementsByTagName('h1')[0] });
+        const title = content.getElementsByTagName('h1')[0];
```
Comment (negarn) [inline src/javascript/app/base/binary_loader.js]: please align the `=` with below

Comment (Khalidxx) [reply src/javascript/app/base/binary_loader.js]: I removed the title 😅 I guess it looks better without

**Pull request merged**